src/app/widget-builder/components/page-list/page-list.component.ts

```diff
@@ -319,8 +319,12 @@ export class PageListComponent implements OnInit {
     this.topbarService.setBackButton(
       new BackButton(
         BackButton.TYPE_LINK,
-        'TOPBAR_BACK_BUTTON_LABEL_PROJECTAANVRAGEN',
-        environment.projectaanvraagDashboardUrl
+        environment.platformIsLive
```

Comment:
This might be better in another PR, but I think we should remove the variable `platformIsLive`, because it is live in production & we will not switch back to `projectaanvraag`.